### PR TITLE
Changed database.

### PR DESCRIPTION
**Pull request by gajrajgchouhan**
#133 Switched to Sqlite and updated requirements.txt, contributing guidelines.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,4 @@
 asgiref==3.2.10
 Django==3.1.2
-Pillow==7.2.0
```
Comment (harnaman-hk) [inline requirements.txt:3]: Don't remove Pillow. It'll be used for images.

Comment (gajrajgchouhan) [reply requirements.txt:3]: Added.

**Comment by harnaman-hk** [comment]
Looks good. I think a rebase is required and then we can merge it.

**Review by harnaman-hk — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,71 +2,36 @@
...
 
-    3. Install required modules using `pip3 install -r requirements.txt`
+    3. Install required modules using `pip3 install -r requirements.txt`.
 
-5.  Store your secrets by adding them at the end of `venv/bin/activate`. They will be set whenever you run the virtual environment.
+4.  Store your secrets by adding them at the end of `venv/bin/activate`. They will be set whenever you run the virtual environment.
 
-    ```
+    ```bash
     export SECRET_KEY=''
```
Comment (harnaman-hk) [inline CONTRIBUTING.md:22]: We need slightly different instruction for windows here as export doesn't work in windows. Add:
set SECRET_KEY=""
for windows users

Comment (gajrajgchouhan) [reply CONTRIBUTING.md:22]: Will do.

**Comment by LalitNM** [comment]
@harnaman-hk should I merge this one?

**Review by harnaman-hk — APPROVED** [review]

**Comment by harnaman-hk** [comment]
@gajrajgchouhan rebase!

**Comment by LalitNM** [comment]
> @gajrajgchouhan rebase!

For next pull request:sweat_smile:

**Pull request merged**